### ISSUE TITLE
[Feature Request] Support `Alt/Super` + `Shift` + `Tab` in Window Switcher

### ISSUE THREAD

**Issue by bbb651** [pop-os/cosmic-launcher#173]
Currently you can only cycle forwards through the list of windows. Using <kbd>shift</kbd> to cycle in the opposite direction is supported pretty much everywhere window/tab switching is supported (other desktops, browsers, code editors, terminals, etc.) and would massively improve usability.

**Comment by git-f0x** [comment]
Completely unrelated, but how did you put shift into that box?

**Comment by bbb651** [comment]
GFM (GitHub Flavored Markdown) supports the `<kbd>` html tag.
You can [do some fun things](https://github.com/zed-industries/zed/issues/14830#issuecomment-2240949169) with it.

**Comment by skylerberg** [comment]
There are PRs for all the parts of this feature that are needed. For convenience, I'm list them all here:

- https://github.com/pop-os/cosmic-launcher/pull/183
- https://github.com/pop-os/cosmic-settings-daemon/pull/49
- https://github.com/pop-os/cosmic-comp/pull/807
- https://github.com/pop-os/cosmic-settings/pull/675

**Comment by skylerberg** [comment]
Yep, I did get it to work locally. Great to see that there is another PR up that covers this change and ordering by recency :)

**Comment by pwall2222** [comment]
Pretty sure #225 also closes this issue!
